packages/core/src/facade/f-univer.ts

```diff
@@ -162,4 +163,8 @@ export class FUniver extends FBase {
     newBlob(): FBlob {
         return this._injector.createInstance(FBlob);
     }
+
+    getUserManager(): FUser {
+        return this._injector.createInstance(FUser);
```

Comment:
If the method's name is `getUserManager`, the Facade class should be `FUserManager` as well.